lib/api/BackbeatAPI.js

```diff
@@ -21,11 +21,12 @@ const { getSortedSetKey, getSortedSetMember } =
 const monitoringClient = require('../clients/monitoringHandler').client;
 
 const {
-    redisKeys,
+    redisKeys: crrRedisKeys,
```

Comment:
Is the  name `crrRedisKeys` updated in the `/../../extensisons/replication/constants` file?

Reply:
@JianqinWang  It's the opposite. Because of object destructuring he is assigning the value of redisKeys to a constant crrRedisKeys.

Reply:
my bad, thanks for clarifyinh